source/api/image/3.0/index.md

```diff
@@ -477,92 +477,126 @@ Servers should also support [CORS][cors-response] on Image Information responses
 
 ### 5.2. Technical Properties
 
-| Technical Property   | Required? | Description |
+The JSON response has several technical properties described in the table below that describe the available functionality for the image content.
```

Comment:
would remove "described in the table below " because that will be obvious from layout, and "described...that describe" is ponderous to read

Reply:
👍